src/languages/lsl.js

```diff
@@ -31,29 +31,29 @@ export default function(hljs) {
         className: 'literal',
         variants: [
             {
-                begin: '\\b(?:PI|TWO_PI|PI_BY_TWO|DEG_TO_RAD|RAD_TO_DEG|SQRT2)\\b'
+                begin: '\\b(PI|TWO_PI|PI_BY_TWO|DEG_TO_RAD|RAD_TO_DEG|SQRT2)\\b'
```

Comment:
There are speed optimizations believe it or not the benchmark was faster without the ":" to mark it as a non capturing group.